WINDOWS.md

```diff
@@ -1210,6 +1210,50 @@ pip freeze
 ```
 
 
+
+## Configuring Jupyter Notebook to open in your browser
+
+Let's generate the configuration file for **Jupyter Notebook**...
+
+``` bash
+jupyter notebook --generate-config
+```
+
+⚠️ Please copy the path returned by the previous command.
+
+We will now edit the generated Jupyter configuration file:
+
+``` bash
+code /home/gmanchon/.jupyter/jupyter_notebook_config.py # ⚠️ replace this path with your own
```

Comment:
```suggestion
code $HOME/.jupyter/jupyter_notebook_config.py
```

Reply:
@dmilon commit gratefully accepted, it works perfectly 🙏 Also the GitHub action seems to have ran the `ruby build.rb`automatically 👍